client/me/connected-application-item/index.jsx

```diff
@@ -120,116 +118,100 @@ module.exports = React.createClass( {
 
 		if ( message ) {
 			return (
-				<li className="connected-application-item__connection-detail">
-					<strong className="connected-application-item__connection-detail-title">
+				<div>
+					<h2>
 						{ this.translate( 'Access Scope' ) }
 						{ this.renderAccessScopeBadge() }
-					</strong>
+					</h2>
 
-					<span className="connected-application-item__connection-detail-description" >
+					<p className="connected-application-item__connection-detail-description" >
 						{ message }
-					</span>
-				</li>
+					</p>
+				</div>
 			);
 		}
 	},
 
 	renderDetail: function() {
-		var connection = this.props.connection;
 		if ( this.props.isPlaceholder ) {
 			return;
 		}
 
 		return (
-			<div className="connected-application-item__content">
-				<p className="connected-application-item__content-description">
-					{ connection.description }
+			<div>
+				<h2>
+					{ this.translate( 'Application Website' ) }
+				</h2>
```

Comment:
This isn't a big deal, but I prefer squashing this into a single line for clarity:

```js
<h2>{ this.translate( 'Application Website' ) }</h2>
```